test/parallel/test-util-inspect.js

```diff
@@ -1950,6 +1950,81 @@ assert.strictEqual(util.inspect('"\'${a}'), "'\"\\'${a}'");
   );
 });
 
+// Verify that classes are properly inspected.
+[
+  // The whitespace is intentional.
+  // eslint-disable-next-line no-multi-spaces
+  [class   { }, 'class (anonymous) {}'],
+  [class extends Error { log() {} }, 'class (anonymous) extends Error {}'],
+  [class A { constructor(a) { this.a = a; } log() { return this.a; } }],
+  [class
+  // Random comments are part of the stringified result
```

Comment:
Is this a typo? Shouldn't it say comments are *not* part of the stringified result?

Reply:
I think he wants to test this regex, but this comment doesn't work 

```js
const stripCommentsRegExp = /(\/\/.*?\n)|(\/\*(.|\n)*?\*\/)/g;
```

https://github.com/nodejs/node/pull/32332/files#diff-91f6ca3fae2551d5959872b74c18946aR172

Reply:
I updated the comment to better reflect that it's only about the `toString` result (it includes comments).